src/nationalcareers_toolkit/assets/src/frontend/sass/compui/dfc-app-pages.scss

```diff
@@ -16,149 +16,510 @@ div.dfc-app-pages-alignment-justify {
+                font-size: 16px;
+                line-height: 1.25;
+            }
+
+            &:hover, &:active, &:focus, &:visited:focus {
+                outline: 3px solid transparent;
+                color: #0b0c0c;
+                background-color: #fd0;
+                box-shadow: 0 -2px #fd0,0 4px #0b0c0c;
+                text-decoration: none;
+                color: #0b0c0c !important;
+            }
+
+            .gem-c-share-links__link-icon {
+                position: absolute;
+                top: 0;
+                left: 0;
+                width: 32px;
+                height: 32px;
+                vertical-align: top;
+            }
+        }
+    }
 }
+
 .gem-c-share-links--stacked {
-  .gem-c-share-links__list-item {
-      display: block
-  }
+    .gem-c-share-links__list-item {
+        display: block
+    }
 }
+
 .direction-rtl .gem-c-share-links__link {
-  display: inline-block;
-  margin-right: 0;
-  margin-left: 30px
+    display: inline-block;
+    margin-right: 0;
+    margin-left: 30px
 }
 
 .direction-rtl .gem-c-share-links__list-item {
-  padding-left: 10px;
-  padding-right: 42px
+    padding-left: 10px;
+    padding-right: 42px
 }
 
 .direction-rtl .gem-c-share-links__link-icon {
-  left: auto;
-  right: 0
+    left: auto;
+    right: 0
 }
 
 .brand--ncs {
-  .brand__color {
-      &:link, &:visited, &:active {
-          color: #207e77;
-      }
-      &:hover,&:focus {
-          color: #0b0c0c;
-      }
-  }
+    .brand__color {
+        &:link, &:visited, &:active {
+            color: #207e77;
+        }
+
+        &:hover, &:focus {
+            color: #0b0c0c;
+        }
+    }
 }
 
 @media print {
-  .gem-c-share-links__link {
-      font-family: sans-serif
-  }
+    .gem-c-share-links__link {
+        font-family: sans-serif
+    }
 }
 
 @media print {
-  .gem-c-share-links__link {
-      font-size: 14pt;
-      line-height: 1.2
-  }
+    .gem-c-share-links__link {
+        font-size: 14pt;
+        line-height: 1.2
+    }
 }
 
 .gem-c-share-links--columns {
-  .gem-c-share-links__list {
-      display: grid;
-  }
-  &:after {
-      content: "";
-      display: block;
-      clear: both
-  }
-  .gem-c-share-links__list-item {
-      float: left;
-      min-width: 150px
-  }
-  .gem-c-share-links__link {
-      margin: 0
-  }
-}
-
-.govuk-full-width-container > .govuk-grid-row{
-
-  .govuk-grid-column-full{
-    padding: 0px;
-  }
-
-  margin: 0 auto;
-}
\ No newline at end of file
+    .gem-c-share-links__list {
+        display: grid;
+    }
+
+    &:after {
+        content: "";
+        display: block;
+        clear: both
+    }
+
+    .gem-c-share-links__list-item {
+        float: left;
+        min-width: 150px
+    }
+
+    .gem-c-share-links__link {
+        margin: 0
+    }
+}
+
+.govuk-full-width-container > .govuk-grid-row {
+
+    .govuk-grid-column-full {
+        padding: 0px;
+    }
+
+    margin: 0 auto;
+}
+
+.hods-pagination__navigation__item {
```

Comment:
@defc0nONE just to confirm are these changes meant to be in pages or should they be in fac?

Reply:
This looks like it's unique to FAC so shall we move it to there?

Reply:
This looks like it's unique to FAC so shall we move it to there?